### PR TITLE
Added more examples for platform resource protection

### PR DESCRIPTION
**Pull request by pscheri**
*Issue #, if available:*

*Description of changes:* Added a few more examples protecting important services like Route53, CloudHSM and AWS Orgs. Also made a small modification on the KMS policy, adding a few more restrictions for unrecoverable objects.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Protect-cloud-platform-resource/Deny-leave-organization.json
Comment (SwaraGandhi) [inline Protect-cloud-platform-resource/Deny-leave-organization.json:1]: This policy is already part of the Privileged access controls. Can we remove it from here or do you think it should be mentioned in "Protect cloud platform resource section".

File: Privileged-access-controls/Deny-unwarranted-IAM-federations-creation.json
Comment (SwaraGandhi) [inline Privileged-access-controls/Deny-unwarranted-IAM-federations-creation.json:1]: Remove  "Sid": "IAM" to help with character space against SCP quota limits

Comment (SwaraGandhi) [reply Privileged-access-controls/Deny-unwarranted-IAM-federations-creation.json:1]: Add -and modification to the file name to match the actions listed

File: Protect-cloud-platform-resource/Protect-cloud-platform-resource.md
Comment (SwaraGandhi) [inline Protect-cloud-platform-resource/Protect-cloud-platform-resource.md:1]: remove | [Deny leaving the AWS Organization](Deny-leave-organization.json) |Restrict the action of leaving the AWS Organization, this protection aims to protect the AWS Accounts against account takeover actions.|

File: Sensitive-data-protection/Deny-stateful-services-deletion.json
Comment (SwaraGandhi) [inline Sensitive-data-protection/Deny-stateful-services-deletion.json:1]: remove sid

File: Protect-cloud-platform-resource/Deny-leave-organization.json
Comment (pscheri) [reply Protect-cloud-platform-resource/Deny-leave-organization.json:1]: Couldn't find this action in other files, leaving it there for now.

File: Privileged-access-controls/Deny-unwarranted-IAM-federations-creation.json
Comment (pscheri) [reply Privileged-access-controls/Deny-unwarranted-IAM-federations-creation.json:1]: resolved

**Pull request closed without merging**